src/d3d/lfx.cpp

```diff
@@ -0,0 +1,42 @@
+#include "lfx.h"
+
+#include <windows.h>
+#include "../util/util_string.h"
+#include "../util/util_log.h"
+
+namespace dxvk {
+    Lfx::Lfx() {
+        const auto lfxModuleName = "latencyflex_wine.dll";
```

Comment:
I agree that having a unified dll name helps. But since this involves a breaking change, I would like to have `latencyflex_wine.dll` as a fallback when we get this released (it can be removed once enough time has passed for the users to update). Does it sound good?

I will update the LatencyFleX side with `latencyflex_layer.dll` becoming the primary bridge, and `latencyflex_wine.dll` as a pure (no native code) export forwarder. This involves a few build system changes, which I'll try to test out in the next few days.

Reply:
Yes, that sounds like a sensible approach, I'm fine with looking for `latencyflex_layer.dll` and retrying with `latencyflex_wine.dll` for a limited time. Please also add a comment (incl. an issue or commit in your repo as a reference?) why the fallback is needed.

Reply:
I added a comment (`// Try fallback entrypoints. These were used by an older version of LatencyFleX.`) but it might deserve more explanation or links for better tracking. Not sure if an issue or a commit link is the best, let me address that in a later revision...

Reply:
I added a link to the commit too.